metadata/virtual_container_host.go

```diff
@@ -36,13 +37,18 @@ type VirtualContainerHostConfigSpec struct {
 	// The base config for the appliance. This includes the networks that are to be attached
 	// and disks to be mounted.
 	// Networks are keyed by interface name
-	ExecutorConfig
-
-	// The set of components to launch
-	ComponentConfig []SessionConfig
-
+	ExecutorConfig `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
Good question. But looks like they are using this version all of the world. @hickeng for comments on it.